lzma-sys/xz-5.2.3/src/liblzma/rangecoder/range_common.h

```diff
@@ -14,10 +14,7 @@
 #ifndef LZMA_RANGE_COMMON_H
 #define LZMA_RANGE_COMMON_H
 
-#ifdef HAVE_CONFIG_H
-#	include "common.h"
-#endif
-
+#include "common.h"
```

Comment:
@alexcrichton this change is actually necessary to get `liblzma` to build on its own. Do you want me to commit a patch to make it easy to re-fix on upgrades?

Defining `HAVE_CONFIG_H` is an okay-ish substitute, we'd have to write a `config.h` with the defines it's expecting.

Reply:
Hm I wonder, in the interest of eventually using a submodule, could we avoid this? We could write an empty `config.h` to get included, right?

Reply:
Yeah, I figured that'd be the preferred approach, but if we're gonna have to write a `config.h`, we might as well stick all the relevant defines in there anyway.